### PR TITLE
Check if figure has 'data' member set

### PR DESCRIPTION
**Pull request by valentijnnieman**
This fixes the bug described in #223

### CODE REVIEW

**Comment by chriddyp** [comment]
:dancer:

**Comment by valentijnnieman** [comment]
The tests keep failing on CircleCI (but work locally) and I'm having no luck debugging it!

**Comment by valentijnnieman** [comment]
@T4rk1n Do you have any idea why these tests are failing on CircleCI? You mentioned something about tests last friday, so I thought maybe you know what's going on here :-)

**Comment by T4rk1n** [comment]
Looks good to me apart from the failing test.

@valentijnnieman The test work locally. I think the `wait_for_element_by_id`  is broken.

**Comment by T4rk1n** [comment]
There's something strange with the test, I tried to change the find element method and add more delay, but it can never find it.

**Comment by valentijnnieman** [comment]
@T4rk1n Yeah, I couldn't figure it out either. Thanks for trying though!

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.23.1]
+### Fixed
+- Fixed bug where Graph wouldn't render if figure.data wasn't set (#223)
```
Comment (chriddyp) [inline CHANGELOG.md]: Similarly here - let's use the full link to the issue [#223](https://github.com/plotly/dash-core-components/issues/223)

File: test/test_integration.py [hunk 2]
```diff
@@ -528,3 +533,22 @@ def update_graph(n_clicks):
...
+        app = dash.Dash(__name__)
+
+        app.layout = html.Div([
+            dcc.Graph(id='graph', figure={
+            })
+        ])
+
+        self.startServer(app=app)
+
+        graph_rendered = WebDriverWait(self.driver, 10).until(
```
Comment (chriddyp) [inline test/test_integration.py]: Have you tried bumping this up to like, 20?

Comment (T4rk1n) [reply test/test_integration.py]: Yes, didn't change, should put a percy snapshot before the wait so we can see if the graph is actually there.

**Comment by chriddyp** [comment]
Maybe lets try rebasing and seeing if the circle issue is fixed?

**Comment by valentijnnieman** [comment]
This was fixed in #285, closing this.

**Pull request closed without merging**